src/cfclient_evaluator.erl

```diff
@@ -154,45 +157,156 @@ search_rules_for_inclusion([], _) -> not_found.
 -spec is_rule_included_or_excluded(Clauses :: list(), Target :: target()) -> true | false.
 is_rule_included_or_excluded([Head | Tail], Target) ->
   case maps:get(op, Head, false) of
-    <<"segmentMatch">> ->
+    ?SEGMENT_MATCH_OPERATOR ->
       %% At present there is only ever one element in values, so we get the head.
       GroupName = hd(maps:get(values, Head, false)),
       CachePid = cfclient_cache_repository:get_pid(),
       Group = cfclient_cache_repository:get_from_cache({segment, GroupName}, CachePid),
-      TargetIdentifier = maps:get(identifier, Target),
-      search_group(excluded, TargetIdentifier, Group);
+      search_group(excluded, Target, Group);
     _ -> is_rule_included_or_excluded(Tail, Target)
   end;
 is_rule_included_or_excluded([], _) -> false.
 
 %% Parses Group Rules for the different rule types.
--spec search_group(RuleType :: atom(), TargetIdentifier :: binary(), Group :: map()) -> included | excluded | false.
-search_group(excluded, TargetIdentifier, Group) ->
+-spec search_group(RuleType :: atom(), Target :: binary(), Group :: map()) -> included | excluded | false.
+search_group(excluded, Target, Group) ->
+  TargetIdentifier = maps:get(identifier, Target),
   case search_group_rules(TargetIdentifier, maps:get(excluded, Group, [])) of
     true ->
       excluded;
     false ->
-      search_group(included, TargetIdentifier, Group)
+      search_group(included, Target, Group)
+  end;
+search_group(included, Target, Group) ->
+  TargetIdentifier = maps:get(identifier, Target),
+  case  search_group_rules(TargetIdentifier, maps:get(included, Group, [])) of
```

Comment:
Nit pick, but extra space here #OCD.

Reply:
Ran the formatter, tidied this up and a few other lines.